### DISCUSSION TITLE
Is it possible to apply styles to methods in a UML diagram?

### DISCUSSION THREAD

**Discussion by gooseboi** [terrastruct/d2#2106] [Q&A]
Like the question says: can you apply, for instance, an underline or italic style to a method in a UML diagram?

**Comment by alixander** [comment]
I don't understand the question, can you rephrase

**Reply by gooseboi** [reply]
Imagine I have the following UML diagram:
```
Square {
    shape: class

    +side: int

    +Area(): int
    +Perimeter(): int
}
```
Is it possible to, for example, make the name "Area()" appear in bold when rendering the diagram?

**Reply by alixander** [reply]
Oh I understand now.

No it doesn't. But we should allow that: https://github.com/terrastruct/d2/issues/2109

Thank you for the suggestion

**Reply by j-standfast** [reply]
+1
